package.json

```diff
@@ -81,16 +81,16 @@
     "**/graphql-toolkit/lodash": "^4.17.15",
     "**/hoist-non-react-statics": "^3.3.2",
     "**/isomorphic-fetch/node-fetch": "^2.6.1",
-    "**/istanbul-instrumenter-loader/schema-utils": "1.0.0",
+    "**/istanbul-instrumenter-loader/schema-utils": "<=1.0.0",
     "**/load-grunt-config/lodash": "^4.17.20",
     "**/minimist": "^1.2.5",
     "**/node-jose/node-forge": "^0.10.0",
     "**/prismjs": "1.22.0",
     "**/react-syntax-highlighter": "^15.3.1",
     "**/react-syntax-highlighter/**/highlight.js": "^10.4.1",
     "**/request": "^2.88.2",
-    "**/trim": "0.0.3",
-    "**/typescript": "4.0.2"
+    "**/trim": "^0.x",
+    "**/typescript": "3.7.2 - 4.0.2"
```

Comment:
Aren't there multiple potential breaking changes between 3.7 and 4.0?

Reply:
Yes there are many breaking changes between these two versions. 4.0 has changed a lot. In our dashboards, we still have some package like @microsoft/api-extractor@7.7.0 or @microsoft/api-extractor@7.7.0 etc. depends on typescript@3.7.2. So **/typescript in selective resolution can force to use 4.0.2 while also allow other packages use their own typescript version. But this will trigger warnings. This is just a tmp fix to silence the warning. If we want a complete fix, we need to update all the related packages that still depends on typescript@3.7.2

Reply:
Okay, is there a reason we need to silence the warnings then? I would argue that silencing the warnings doesn't necessarily lead to a "clean build", if that is what our goal is.

Reply:
I think we shouldn't silence the warnings. I would see whether we can update the packages that triggered warnings and their dependencies.